### PR TITLE
Reminder to update the Adventure JD on the homepage when changing the version

### PR DESCRIPTION
**Pull request by aurorasmiles**
As requested in https://github.com/PaperMC/papermc.io/pulls#issuecomment-808766783

### CODE REVIEW

File: Spigot-API-Patches/0005-Adventure.patch [hunk 1]
```diff
@@ -20,7 +20,7 @@ index e3744ee042e0426a513c4fdf4abedafe85e31cd2..75b2830340051deb0fa39149e80872d2
 +            <dependency>
 +                <groupId>net.kyori</groupId>
 +                <artifactId>adventure-bom</artifactId>
-+                <version>4.7.0</version> <!-- keep this version in synch with javadoc section below! -->
++                <version>4.7.0</version> <!-- keep this version in sync with javadoc section below! When updating this make sure to update the linked JavaDocs on the homepage as well!-->
```
Comment (kashike) [inline Spigot-API-Patches/0005-Adventure.patch]:
```suggestion
+                <version>4.7.0</version> <!-- keep this version in synch with javadoc section below *and* on the papermc.io repository! -->
```

**Review by kashike — DISMISSED** [review]

File: Spigot-API-Patches/0005-Adventure.patch (hunk 1 above)
Comment (zml2008) [reply Spigot-API-Patches/0005-Adventure.patch]: :( the extra h makes me unhappy

though -- can you extract this into a maven project property so there's only one place in the pom that needs updating? I think that should work in javadoc `links` URLs too, but definitely works in pom versions.

**Review by zml2008 — CHANGES_REQUESTED** [review]

**Comment by aurorasmiles** [comment]
Go home github, you are drunk

File: Spigot-API-Patches/0005-Adventure.patch (hunk 1 above)
Comment (aurorasmiles) [reply Spigot-API-Patches/0005-Adventure.patch]: Done

**Review by jpenilla — APPROVED** [review]

**Review by electronicboy — APPROVED** [review]

**Comment by aurorasmiles** [comment]
@limbo-app ready

**Review by limbo-app — APPROVED** [review]
Pull request automatically approved for merging - a core team member has previously approved this pull request.

**Pull request merged**